optaplanner-core/src/test/java/org/optaplanner/core/config/constructionheuristic/placer/PooledEntityPlacerConfigTest.java

```diff
@@ -16,46 +16,46 @@
 
 package org.optaplanner.core.config.constructionheuristic.placer;
 
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
-import java.util.Arrays;
-
 import org.junit.jupiter.api.Test;
 import org.optaplanner.core.config.heuristic.selector.move.generic.ChangeMoveSelectorConfig;
 import org.optaplanner.core.config.heuristic.selector.value.ValueSelectorConfig;
 import org.optaplanner.core.config.solver.EnvironmentMode;
-import org.optaplanner.core.impl.constructionheuristic.placer.QueuedEntityPlacer;
 import org.optaplanner.core.impl.domain.solution.descriptor.SolutionDescriptor;
 import org.optaplanner.core.impl.heuristic.HeuristicConfigPolicy;
-import org.optaplanner.core.impl.score.buildin.simple.SimpleScoreDefinition;
 import org.optaplanner.core.impl.score.director.InnerScoreDirectorFactory;
-import org.optaplanner.core.impl.testdata.domain.multivar.TestdataMultiVarSolution;
+import org.optaplanner.core.impl.testdata.domain.TestdataEntity;
+import org.optaplanner.core.impl.testdata.domain.TestdataSolution;
 
-public class QueuedEntityPlacerConfigTest {
+public class PooledEntityPlacerConfigTest {
```

Comment:
I presume this is just GitHub getting confused?

Reply:
Short answer: yes; long answer:
during this refactoring, I look in the coverage reports and try to cover the biggest gaps. Often, the gaps are in the build methods, thus the `QueuedEntityPlacerConfigTest` moved to `QueuedEntityPlacerFactoryTest`. On the other hand, the `PooledEntityPlacerConfig` had a coverage gap in its `unfoldNew()`, which lead to introducing a new test class.